### PR TITLE
Use sdk.getpebble.com URLs for the SDK download

### PR DESCRIPTION
**Pull request by matthewtole**
Let me know if I messed up or need to make the change in more places.

### CODE REVIEW

**Comment by cjwilliams** [comment]
Like

:+1:

File: pebble-sdk-legacy.rb [hunk 1]
```diff
@@ -7,7 +7,7 @@ class PebbleSdkLegacy < Formula
   PEBBLE_SDK_SHA256='9eb9dee08c88f192a6403717d8b32d67f49e598efb1b6e530056f008c5a2f465'
 
   homepage 'https://developer.getpebble.com'
-  url "http://assets.getpebble.com.s3-website-us-east-1.amazonaws.com/sdk2/PebbleSDK-#{PEBBLE_SDK_VERSION}.tar.gz"
+  url "https://sdk.getpebble.com/download/#{PEBBLE_SDK_VERSION}/?source=homebrew-legacy"
```
Comment (cjwilliams) [inline pebble-sdk-legacy.rb]: Out of curiosity, should this be pebble.com now or is it still getpebble?

Comment (Katharine) [inline pebble-sdk-legacy.rb]: getpebble.com is correct. 

Also, that trailing slash seems really weird for a file download. Can we drop that?

Comment (matthewtole) [inline pebble-sdk-legacy.rb]: I think we've agreed to keep everything on getpebble.com until the main website is moved. 

> Also, that trailing slash seems really weird for a file download. Can we drop that?

Yes.

**Comment by matthewtole** [comment]
Any other comments @Katharine?

**Comment by Katharine** [comment]
LGTM!

**Pull request merged**